test/sqllogictest/role_attributes.slt

```diff
@@ -0,0 +1,382 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Is this hand-written or taken from Cockroach?

Reply:
I'm not sure where it originates, I copied from another `.slt` in this directory.